library/src/main/java/com/squareup/timessquare/CalendarPickerView.java

```diff
@@ -406,7 +415,8 @@ private void scrollToSelectedMonth(final int selectedIndex, final boolean smooth
         if (smoothScroll) {
           smoothScrollToPosition(selectedIndex);
         } else {
-          setSelection(selectedIndex);
+          scrollToSelectedMonth(selectedIndex, false);
```

Comment:
was this an intentional change?  i think the initial intent here was to just immediately snap to the new position if `!smoothScroll`